app/BlockCategory.php

```diff
@@ -5,7 +5,7 @@ class BlockCategory implements \Dxw\Iguana\Registerable
 {
     public function register()
     {
-        add_filter('block_categories', [$this, 'newBlocksCategory'], 10, 1);
+        add_filter('block_categories_all', [$this, 'newBlocksCategory'], 10, 1);
     }
 
     public function newBlocksCategory($categories)
```

Comment:
Note that we ignore the second parameter here which is the main thing that's changed in the new filter (`WP_Post` -> `WP_Block_Editor_Context`)